build.sh

```diff
@@ -816,6 +819,15 @@ while :; do
         msbuildonunsupportedplatform)
             __msbuildonunsupportedplatform=1
             ;;
+        numproc)
+            if [ -n "$2" ]; then
+              NumProc="$2"
```

Comment:
Could you please change this to __numProc to match other options extracted from the command line? And to match the style of initialization, it would be great to move the extraction of the default value (getting the processor count) from the build_native to the place where all the command line option variables are initialized.